packages/foreman/foreman/foreman.spec

```diff
@@ -1014,8 +1012,8 @@ install -Dpm0644 %{SOURCE6} %{buildroot}%{_sysconfdir}/yum.repos.d/%{name}.repo
 install -Dpm0644 %{SOURCE7} %{buildroot}%{_sysconfdir}/yum.repos.d/%{name}-plugins.repo
 sed "s/\$DIST/$(echo %{?dist} | cut -d. -f2)/g" -i %{buildroot}%{_sysconfdir}/yum.repos.d/%{name}*.repo
 if [[ '%{release}' != *"develop"* ]];then
```

Comment:
Should we change this to just check `[[ %{prerelease} == "develop" ]]`?

Reply:
Ehh, this feels fine still to me.

Reply:
Fair enough.